client/src/document/ingredients/browser-compatibility-table/index.scss

```diff
@@ -150,6 +150,7 @@
   -ms-writing-mode: tb-rl;
   -webkit-writing-mode: vertical-rl;
   writing-mode: vertical-rl;
+  text-orientation: sideways;
```

Comment:
```scss
.bc-head-txt-label {
  left: calc(50% - 0.5rem);
  line-height: 1;
  padding-top: 0.5rem;
  position: relative;
  text-orientation: sideways;
  transform: rotate(180deg);
  white-space: nowrap;
  -ms-writing-mode: tb-rl;
  -webkit-writing-mode: vertical-rl;
  writing-mode: vertical-rl;
}
```

There is an error in the ordering of the attribute values in the stylint, please move the transform up and it will fix the lint.

Reply:
@1ilsang Thank you for your review! I've fixed it. This was caused by me not reading the contribution documentation properly. I will read it properly before contributing.